code/modules/games/cards.dm

```diff
@@ -0,0 +1,472 @@
+
+	set category = "Object"
+	set name = "Deal Multiple Cards"
+	set desc = "Deal multiple cards from a deck."
+	set src in view(1)
+
+	if(usr.incapacitated() || !Adjacent(usr))
+		return
+
+	if(!cards.len)
+		to_chat(usr,"<span class='notice'>There are no cards in the deck.</span>")
+		return
+
+	var/list/players = list()
+	for(var/mob/living/player in viewers(3))
+		if(!player.incapacitated())
+			players += player
+	var/maxcards = max(min(cards.len,10),1)
+	var/dcard = input("How many card(s) do you wish to deal? You may deal up to [maxcards] cards.") as num
+	if(dcard > maxcards)
+		return
+	var/mob/living/M = input("Who do you wish to deal [dcard] card(s)?") as null|anything in players
+	if(!usr || !src || !M || !Adjacent(usr)) return
+
+	deal_at(usr, M, dcard)
+
+/obj/item/weapon/deck/proc/deal_at(mob/user, mob/target, dcard) // Take in the no. of card to be dealt
+	var/obj/item/weapon/cardhand/H = new(get_step(user, user.dir))
+	var/i
+	for(i = 0, i < dcard, i++)
+		H.cards += cards[1]
+		cards -= cards[1]
+		H.parentdeck = src
+		H.concealed = TRUE
+		H.update_icon()
+	if(user==target)
+		user.visible_message("<span class = 'notice'>[user] deals [dcard] card(s) to \himself.</span>")
+	else
+		user.visible_message("<span class = 'notice'>[user] deals [dcard] card(s) to [target].</span>")
+	H.throw_at(get_step(target,target.dir),3,1,H)
+
+
+/obj/item/weapon/deck/attack_self()
+	deckshuffle()
+
+/obj/item/weapon/deck/verb/verb_shuffle()
+	set category = "Object"
+	set name = "Shuffle"
+	set desc = "Shuffle the cards in the deck."
+	set src in view(1)
+	deckshuffle()
+
+/obj/item/weapon/deck/proc/deckshuffle()
+	var/mob/living/user = usr
+	if (cooldown < world.time - 5 SECONDS)
+		cards = shuffle(cards)
+		user.visible_message("<span class = 'notice'>[user] shuffles [src].</span>")
+		playsound(user, 'sound/items/cardshuffle.ogg', 50, 1)
+		cooldown = world.time
+
+
+/obj/item/weapon/deck/MouseDrop(atom/over_object) // Code from Paper bin, so you can still pick up the deck
+	var/mob/M = usr
+	if(M.incapacitated() || !Adjacent(M))
+		return
+
+	if(over_object == M)
+		M.put_in_hands(src)
+
+	else if(istype(over_object, /obj/screen))
+		switch(over_object.name)
+			if("r_hand")
+				if(!remove_item_from_storage(M))
+					M.unEquip(src)
+				M.put_in_r_hand(src)
+			if("l_hand")
+				if(!remove_item_from_storage(M))
+					M.unEquip(src)
+				M.put_in_l_hand(src)
+
+	add_fingerprint(M)
+	usr.visible_message("<span class = 'notice'>[usr] picks up the deck.</span>")
+
+/obj/item/weapon/pack
+	name = "Card Pack"
+	desc = "For those with disposable income."
+
+	icon_state = "card_pack"
+	icon = 'icons/obj/playing_cards.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	var/list/cards = list()
+	var/parentdeck = null // For future card pack that need to be compatible with eachother i.e. cardemon
+
+
+/obj/item/weapon/pack/attack_self(mob/user as mob)
+	user.visible_message("<span class = 'notice'>[name] rips open the [src]!</span>")
```

Comment:
No spacing in spans, `<span class='notice'>`
Also `[user]` not `[name]`
Also you need a second argument for first person on user